jcvi/graphics/base.py

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env python
 # -*- coding: UTF-8 -*-
 
+from os import remove
 import copy
+import logging
```

Comment:
nit: there is a package-wide `logger` that you can use.

```python
from ..apps.base import logger
```

Then use `logger.debug()` below.